addons/mail/mail_followers.py

```diff
@@ -191,7 +191,7 @@ def _notify_email(self, cr, uid, ids, message_id, force_send=False, user_signatu
             return True
 
         # compute email body (signature, company data)
-        body_html = message.body
+        body_html = html_sanitize(context.get('mail_body')) if context.get('mail_body') else message.body
```

Comment:
@rim-odoo 
If 'mail_body' in context available than we have to use html_sanitize.
So, Use <code>html_sanitize(context.get('mail_body')) or message.body</code>